project.clj

```diff
@@ -9,6 +9,10 @@
           :defaults {:doc/format :markdown}
           :include [riddley.walk riddley.compiler]
           :output-dir "doc"}
-  :profiles {:provided {:dependencies [[org.clojure/clojure "1.8.0"]]}}
+  :profiles {:provided {:dependencies [[org.clojure/clojure "1.8.0"]]}
+             :dev {:dependencies [[org.clojure/clojure "1.10.1"]
+                                  [pjstadig/humane-test-output "0.10.0"]]
+                   :injections   [(require 'pjstadig.humane-test-output)
+                                  (pjstadig.humane-test-output/activate!)]}}
   :java-source-paths ["src/riddley"]
-  :javac-options ["-target" "1.6" "-source" "1.6"])
+  :javac-options ["-target" "1.7" "-source" "1.7"])
```

Comment:
I'm using Java 14 locally and it doesn't support compiling to Java 1.6 bytecode. I think dropping 1.6 support makes sense -- [Clojure 1.10+ requires Java 8+](https://github.com/clojure/clojure/blob/master/changes.md) and Clojure 1.5+ requires Java 7 for all features to work out-of-the-box